shell/platform/linux/fl_renderable.h

```diff
@@ -0,0 +1,40 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_LINUX_FL_RENDERABLE_H_
+#define FLUTTER_SHELL_PLATFORM_LINUX_FL_RENDERABLE_H_
+
+#include <gdk/gdk.h>
+
+#include "flutter/shell/platform/embedder/embedder.h"
+
+G_BEGIN_DECLS
+
+G_DECLARE_INTERFACE(FlRenderable, fl_renderable, FL, RENDERABLE, GObject);
+
+/**
+ * FlRenderable:
+ *
+ * An interface for a class that can render views from #FlRenderer.
+ *
+ * This interface is typically implemented by #FlView.
```

Comment:
Should we add to this comment that this interface is used primarily for testing? One might incorrectly assume there are multiple renderable implementations. Just a thought, I don't feel strongly, feel free to skip